--- ktranslatoid.cpp	2009-12-15 20:00:32.000000000 +0200
+++ ktranslatoid.cpp.new	2010-04-13 19:47:20.280459383 +0200
@@ -326,7 +326,8 @@
 
     if (result.contains("dict"))
         dict = result["dict"].toList();
-
+    
+    textToReturn += "<HTML><BODY>";
 
 foreach (QVariant trad , sentences)
     {
@@ -363,6 +364,7 @@
     }
 
     textToReturn += dictionnary;
+    textToReturn += "</BODY></HTML>";
     m_destText->setText(textToReturn);
 }
 
